third_party/cuda/BUILD

```diff
@@ -19,7 +19,7 @@ cc_library(
     name = "cudart",
     srcs = select({
         ":aarch64_linux": [
-            "targets/aarch64-linux/lib/libcudart.so",
+            "targets/aarch64-linux-gnu/lib/libcudart.so",
```

Comment:
I don't think these paths are correct for the stock Jetpack. There is no gnu

Reply:
On Jetson AGX, it was failing. The correct path includes gnu on Jetson AGX target

Reply:
Fixed it